src/components/content/Dashboard/Dashboard.js

```diff
@@ -74,11 +74,9 @@ class Dashboard extends Component {
 		const displayGroups = groups.map( group => {
 			const index = groups.indexOf(group);
 			return (
-				<li className="create-project-thumb">
+				<li className="project-thumb">
```

Comment:
Here, the "list" tag is the parent of the "Link" tag, later it's the opposite, which is it?